src/service_aggregator.py

```diff
@@ -306,17 +312,17 @@ async def check_for_messages(guid, pydantic_kgraph, accumulated_results, num_que
                 async for message in queue_iter:
                     async with message.process():
                         num_responses += 1
-                        logger.debug(f'{guid}: Strider returned {num_responses} out of {num_queries}.')
+                        logger.debug(f"{guid}: Strider returned {num_responses} out of {num_queries}.")
                         jr = process_message(message)
                         if is_end_message(jr):
-                            logger.debug(f'{guid}: Received complete message from multistrider')
+                            logger.debug("Received complete message from multistrider")
```

Comment:
I removed the guid from the logging message since it is redundant with the logging message just above it (line 315).  Maybe switch which line has the guid?

Reply:
I think it's still very useful. The log on 315 doesn't tell us what type of response we got back. If Strider happens to send back a completed message before we've received all `num_queries`, we would have no idea, and that would be very bad. In general, it's nice to look through the logs and match every log with a uid.

Reply:
Yeah, the log is only comprehensible when every message has its guid - otherwise they intermix and it's just too hard to understand who did what

Reply:
I am not sure I agree, but I am not going to quibble over it.  I will add it back.